source/read/specify-documents-to-return.txt

```diff
@@ -0,0 +1,198 @@
+.. _kotlin-sync-specify-documents-to-return:
+
+===========================
+Specify Documents to Return
+===========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: read, paginate, pagination, order, code example
+
+Overview
+--------
+
+In this guide, you can learn how to specify which documents to return from a read
+operation by using the following methods:
+
+- ``limit()``: Specifies the maximum number of documents to return from a query.
+- ``sort()``: Specifies the sort order for the returned documents.
+- ``skip()``: Specifies the number of documents to skip before returning query results.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``restaurants`` collection in the ``sample_restaurants``
+database from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+free MongoDB Atlas cluster and load the sample datasets, see the
+:atlas:`Get Started with Atlas </getting-started>` guide.
+
+The documents in this collection are modeled by the following {+language+} data class:
+
+.. literalinclude:: /includes/read/specify-documents-to-return.kt
+   :start-after: start-data-class
+   :end-before: end-data-class
+   :language: kotlin
+   :copyable:
+
+.. _kotlin-sync-limit:
+
+Limit
+-----
+
+To specify the maximum number of documents returned from a read operation,
+call the ``limit()`` method.
+
+The following example finds all restaurants that have a ``cuisine`` field value
+of ``"Italian"`` and limits the results to 5 documents:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: /includes/read/specify-documents-to-return.kt
+      :start-after: start-limit
+      :end-before: end-limit
+      :language: kotlin
+      :dedent:
+
+   .. output::
+      :visible: false
+
+      Restaurant(name=Philadelphia Grille Express, borough=Brooklyn, cuisine=Italian)
+      Restaurant(name=Isle Of Capri Resturant, borough=Manhattan, cuisine=Italian)
+      Restaurant(name=Marchis Restaurant, borough=Manhattan, cuisine=Italian)
+      Restaurant(name=Crystal Room, borough=Staten Island, cuisine=Italian)
+      Restaurant(name=Forlinis Restaurant, borough=Manhattan, cuisine=Italian)
+
+.. tip::
+
+   The preceding example returns the first five documents returned by the query in
+   no particular order. The following section describes how to return the documents
```

Comment:
I: Technically the docs are returned in [natural order](https://www.mongodb.com/docs/manual/reference/glossary/#std-term-natural-order)

```suggestion
   The preceding example returns the first five documents returned by the query in
   :manual:`natural order </reference/glossary/#std-term-natural-order>`. The following section describes how to return the documents
```